app/core/model/account/errors.go

```diff
@@ -0,0 +1,24 @@
+package account
+
+import "errors"
+
+var (
+
+	// ErrCreating occurs when an untracked error happens when creating a new account
+	ErrCreating = errors.New("an error occurred while creating this account")
+
+	// ErrInvalid throws when the identification provided is an invalid form of ID
+	ErrInvalid = errors.New("the account id provided is invalid. please, try again with a valid one")
```

Comment:
só para deixar mais específico que é de um ID

```suggestion
	ErrInvalidID = errors.New("the account id provided is invalid. please, try again with a valid one")
```

Reply:
Beleza, realmente fica melhor e mais assertivo, como o @RegisMelgaco também pontuou. Irei alterar isso